frontend/src/components/page-lifeGroups/fiveEs.js

```diff
@@ -0,0 +1,156 @@
+import * as React from "react";
+import { Circle } from "../svgs/index";
+import { PrimaryButtonLink } from "../Button";
+
+const fiveEInfo = [
+  {
+    word: "Evangelism",
+    text1Bold: "We value ",
+    text1:
+      "God's desire for all people to have a personal and life-transforming relationship with Jesus Christ.",
+    text2:
+      ", we believe that every Christian is called to invest in the process of sharing the Gospel to those in their sphere of influence, their community, and to the nations.",
+    text2Bold: "Therefore",
```

Comment:
here should be the update: 
```suggestion
    text: "<p><span>We value</span>  God's desire for all people to have a personal and life-transforming relationship with Jesus Christ.</p><p><span>Therefore</span>, we believe that every Christian is called to invest in the process of sharing the Gospel to those in their sphere of influence, their community, and to the nations.",
```